doc/source/examples/resolved-cfd-dem/cylinder-with-sharp-interface/cylinder-with-sharp-interface.rst

```diff
@@ -0,0 +1,172 @@
+	# Boundary Conditions
+	#---------------------------------------------------
+
+	subsection boundary conditions
+	set number                  = 3
+	   
+	    subsection bc 0
+	set id = 0
+		set type              = function
+		subsection u
+		    set Function expression = 1
+		end
+		subsection v
+		    set Function expression = 0
+		end
+		subsection w
+		    set Function expression = 0
+		end
+	    end
+	    subsection bc 1
+	set id = 2
+		set type              = slip
+	    end
+	    subsection bc 2
+	set id = 3
+		set type              = slip
+	    end
+	end
+	
+The initial condition has been modified compared to the initial solution proposed in `Example 3 <../../incompressible-flow/2d-flow-around-cylinder/2d-flow-around-cylinder.html>`_. We use the following initial condition to ensure that the particle's boundary condition is satisfied.
+
+.. code-block:: text
+
+	#---------------------------------------------------
+	# Initial condition
+	#---------------------------------------------------
+	subsection initial conditions
+	    set type = nodal
+	    subsection uvwp
+		    set Function expression = 0; 0; 0
+	    end
+	end
+	
+The only thing that is left to define is the immersed boundary. At this point, the immersed boundary tool only allows the representation of spherical objects. This means the immersed boundary can represent a cylinder in 2D and a spherical shell in 3d. We refer to the parameter definition of the immersed boundary in `this <../../../parameters/resolved_cfd-dem/resolved_cfd-dem.html>`_ section of the documentation.
+
+In this case, we want to define a cylindrical boundary of radius 0.5 center at (8,8) that has no velocity.
+
+.. code-block:: text
+
+	# --------------------------------------------------
+	# IB particules
+	#---------------------------------------------------
+        subsection particles
+                set number of particles = 1
+                set stencil order = 2
+                set refine mesh inside radius factor = 0.8
+                set refine mesh outside radius factor = 1.2
+                set initial refinement = 0
+                set integrate motion = false
+                set assemble Navier-Stokes inside particles = false
+            subsection particle info 0
+                subsection position
+                    set Function expression =8;8
+                end
+                subsection velocity
+                    set Function expression =0;0
+                end
+                    set radius = 0.5
+            end
+        end
+
+* The ``number of particles`` is set to one as we only want one cylinder.
+
+* The ``stencil order`` is set to 2 as this is the highest order we can use, and it won't lead to Runge instability.
+
+* The ``refine mesh inside radius factor`` is set to 0.8. This will create a mesh refinement around the particle that avoids having hanging nodes used in the calculation and help ensure a small enough mesh around the cylinder.
```

Comment:
```suggestion
* The ``refine mesh inside radius factor`` is set to 0.8. This will create a mesh refinement around the particle that avoids having hanging nodes in the calculation and help ensure an adequately fine mesh around the particle.
```